ui/components/app/currency-input/currency-input.js

```diff
@@ -44,13 +44,16 @@ export default function CurrencyInput({
 
   const [isSwapped, setSwapped] = useState(false);
   const [newHexValue, setNewHexValue] = useState(hexValue);
+  const [newFeatureSecondary, setNewFeatureSecondary] = useState(
```

Comment:
Can you give a brief comment what this state does?

Reply:
@segun This state use `featureSecondary` prop to show what currency is displayed when user click on `Send` button. When user press the swap (^v) icon `newFeatureSecondary` changed and display the estimated value of the new input type.